alacritty/src/display.rs

```diff
@@ -351,6 +372,11 @@ impl Display {
         let physical = PhysicalSize::new(self.size_info.width as u32, self.size_info.height as u32);
         self.window.resize(physical);
         self.renderer.resize(&self.size_info);
+
+        // Enable subpixel anti-aliasing.
+        #[cfg(target_os = "macos")]
+        let use_thin_strokes = config.font.use_thin_strokes();
+        Self::enable_font_smoothing(use_thin_strokes);
```

Comment:
The idea behind calling this function here is that when we do changes to the config we want to make sure if it affects the `use_thin_strokes` we update that to the new value. Otherwise we would have to restart Alacritty twice in order to make it apply.